src/app/ui/app/App.tsx

```diff
@@ -63,12 +64,22 @@ const App = ({
           <WelcomeScreen closed={!enabled || welcomeScreenClosed!} />
         </Box>
       </Slide>
-      <WidgetContainer
-        welcomeScreenClosed={!enabled || welcomeScreenClosed!}
-        className="widget-container"
+      <Stack
+        display="flex"
```

Comment:
Should not we create a dedicated component for this

Reply:
Yes, maybe a styled-component for this?